E-Commerce-API/app.js

```diff
@@ -1,8 +1,175 @@
 const express = require("express");
 const app = express();
+var bodyParser = require("body-parser");
 // Your code to run the server should go here
 // Don't hardcode your DB password in the code or upload it to GitHub! Never ever do this ever.
 // Use environment variables instead:
 // https://www.codementor.io/@parthibakumarmurugesan/what-is-env-how-to-set-up-and-run-a-env-file-in-node-1pnyxw9yxj
+app.listener = app.listen(3000, function () {
+  console.log(`Server is listening on port ${3000}`);
+});
+const { Pool } = require("pg");
 
+const db = new Pool({
+  user: "mickeyhaile", // replace with you username
+  host: "localhost",
+  database: "cyf_ecommerce",
+  password: "",
```

Comment:
I don't see where you use env vars to add this in? You should have `process.env.DB_PASSWORD` or something somewhere here.